src/CfgMgr32/CfgMgr32+SafeNotificationHandle.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using Microsoft.Win32.SafeHandles;
+
+    /// <content>
+    /// Contains the <see cref="SafeNotificationHandle"/> nested enum.
+    /// </content>
+    public static partial class CfgMgr32
+    {
+        public class SafeNotificationHandle : SafeHandleZeroOrMinusOneIsInvalid
```

Comment:
Did you find documentation to suggest that `0` and `-1` are both invalid handle values?

Reply:
Nope. I did observe that the handle appears to be `0` when `CM_Register_Notification` returns `CR_FAILURE`, so I'll keep it at `0` for now. I added a unit test.

Reply:
I guess the win API header files define `INVALID_HANDLE_VALUE` to be `-1` so perhaps it's a safe assumption that -1 will never be valid.

Reply:
OK - went back to `SafeHandleZeroOrMinusOneIsInvalid`, then.